### DISCUSSION TITLE
"Change class" and/or "Rebirth" services

### DISCUSSION THREAD

**Discussion by hofik19** [chromiecraft/chromiecraft#4040] [Suggestions]
Hello!

I would like to suggest additional ChromiePoints paid service that would let players change their class. Service could be limited to character race or it could be done in form of "Rebirth" package that would combine faction, race, name and class change services in to one.
I don't know if this kind of change is technically possible as I'm not familiar with AzerothCore programming.  

Why would anyone want to change class?
Because we are playing on WotLK 3.3.5a our progress on one character in form of achievements, mounts collection, reputation, completed quests and so on are only accessible on one character where we have done it. All this is not account-wide (except transmogs). This creates a situation where players can be discouraged to try (and test for bugs) different classes when they already have one leveled and "achieved" character. Making another one would mean spending additional tens or hundreds of hours to get it to the same level of progression. I am aware, that making additional progress account-wide is probably very difficult and time consuming but it would also solve this issue. That's why, as alternative, I'm suggesting option to change your character class. This would allow to preserve all progress (except gear ofc) on one character and let players like me, have everything they've achieved accessible in one place and try out another class a the same time.

**Comment by BeefSteaker** [comment]
While I'm not entirely against it, I imagine it would be something quite hard to program as it's not a service bliZard offered at the time, unlike the other changes
